tests/test_post.py

```diff
@@ -105,8 +132,10 @@ def test_post_prior(tmpdir, temperature):
             new_mean, new_cov = mpi.share()
         # Noisier with higher temperature
         atol, rtol = (1e-8, 1e-5) if temperature == 1 else (5e-4, 1e-3)
```

Comment:
Well spotted. This was there since a while ago, recent fixes make it unnecessary. Removing. (If anything, if detempering loaded chains we lose some precision due to the ".8g" txt formatting.)